pyvelox/pyvelox.h

```diff
@@ -504,6 +519,48 @@ static void addVectorBindings(
             std::move(baseVector),
             PyVeloxContext::getSingletonInstance().pool());
       });
+
+  m.def(
+      "row_vector",
+      [](std::vector<std::string>& names,
+         std::vector<VectorPtr>& values,
+         std::function<bool(vector_size_t /*row*/)> isNullAt = nullptr) {
+        auto setNulls =
+            [](const VectorPtr& vector,
+               std::function<bool(vector_size_t /*row*/)> isNullAt) {
+              for (vector_size_t i = 0; i < vector->size(); i++) {
+                if (isNullAt(i)) {
+                  vector->setNull(i, true);
+                }
+              }
+            };
+
+        std::vector<std::shared_ptr<const Type>> childTypes;
+        childTypes.resize(values.size());
+        for (int i = 0; i < values.size(); i++) {
+          childTypes[i] = values[i]->type();
+        }
+        auto rowType = ROW(std::move(names), std::move(childTypes));
+        const size_t vectorSize = values.empty() ? 0 : values.front()->size();
```

Comment:
Its possible to have legal vectors with child size < row size , if the rest of the row vectors consists of trailing nulls.

Reply:
Example consider a Row with one child of size 10 and parent of size 20. Here this is legal if the parent has nulls from index 10 to 19.

Reply:
I understand, then how should we set the `vectorSize` would it be accurate to assume it as the size of child elements?

Reply:
The vector size will be size of row (thus 20) , the child sizes can be different though.

Reply:
In this case for simplicity now we can enforce ensuring that all children and parent are of same size.

Reply:
I updated the code based on this discussion.